doc/modules/ROOT/pages/usage/tls.adoc

```diff
@@ -0,0 +1,56 @@
+= TLS (Transport Layer Security) support
+
+The standard way to securely access a nREPL server has been
+to use `ssh` with port forwarding, and letting the nREPL server only bind to `127.0.0.1`.
+
+Sometimes however `ssh` may not be available, and thus you need to
```

Comment:
I'd expand a little bit on this section - e.g. mention that the connection over TLS will be encrypted and require a certain key to be established. We can't make too many assumptions about the background of the readers of the docs.